src/pages/representatives.js

```diff
@@ -192,11 +152,17 @@ const RepresentativesPage = ({ data }) => {
       <section className="section" css={{ background: "#eeeeee" }}>
         <div className="book">
           <div className="page leftPage">
-            <h1 css={{ ...cssH1, margin: "1rem 0 0 0", fontSize: "3.6rem" }}>สมาชิกสภาผู้แทนราษฎรไทย ชุดที่ 25</h1>
+            <h1 css={{ ...cssH1, margin: "1rem 0 0 0", fontSize: "3.6rem" }}>
+              สมาชิกสภาผู้แทนราษฎรไทย ชุดที่ 25
+            </h1>
             <h2 style={{ ...cssEngTitle }}>25th House of Representative</h2>
             <h2 style={{ ...cssEngTitle }}>About</h2>
             <p style={{ ...cssPageP }}>
-            เป็นส่วนหนึ่งของฝ่ายนิติบัญญัติ ที่ได้มาจากการเลือกตั้งทั่วไป พ.ศ. 2562 ประกอบด้วยสมาชิก (ส.ส.) 500 คนตามระบบจัดสรรปันส่วนผสม โดย 350 คนเป็นผู้แทนเขต และอีก 150 คนมาจากระบบบัญชีรายชื่อ ฝ่ายรัฐบาลเป็นรัฐบาลผสมพรรคพลังประชารัฐ ส่วนฝ่ายค้านคือพรรคเพื่อไทยและพรรคร่วมฝ่ายค้านอีก 6 พรรค
+              เป็นส่วนหนึ่งของฝ่ายนิติบัญญัติ ที่ได้มาจากการเลือกตั้งทั่วไป พ.ศ.
+              2562 ประกอบด้วยสมาชิก (ส.ส.) 500 คนตามระบบจัดสรรปันส่วนผสม โดย 350
+              คนเป็นผู้แทนเขต และอีก 150 คนมาจากระบบบัญชีรายชื่อ
+              ฝ่ายรัฐบาลเป็นรัฐบาลผสมพรรคพลังประชารัฐ
+              ส่วนฝ่ายค้านคือพรรคเพื่อไทยและพรรคร่วมฝ่ายค้านอีก 6 พรรค
             </p>
```

Comment:
Maybe at some point the future, we should separate this text from the template.  For now, it's ok.

@rapee what do you think?

Reply:
Yep, this description should be sourced from data. In the next update.